sphinx_search/extension.py

```diff
@@ -17,9 +18,9 @@ def copy_asset_files(app, exception):
 def setup(app):
 
     app.connect('build-finished', copy_asset_files)
-    
-    for f in CUSTOM_ASSETS_FILES:
-        if f.endswith('.min.js'):
-            app.add_js_file(f)
-        if f.endswith('.min.css'):
-            app.add_css_file(f)
+
+    for file in CUSTOM_ASSETS_FILES:
+        if file.endswith('.min.js'):
+            app.add_js_file(file)
+        if file.endswith('.min.css'):
```

Comment:
For now, I'd like to use the unminified version here, so it's easier to debug.

Reply:
Updated the PR.

Reply:
I wonder if we should have a setting that specifies minified or non-minified files here? We can ship like it is, but I almost always prefer full versions when doing dev.